dask_saturn/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .core import SaturnCluster
+from .core import describe_sizes, list_sizes
```

Comment:
Sorry with the one above as well

Reply:
ah ok. done: https://github.com/saturncloud/dask-saturn/pull/27/commits/11cf702ff2906f220068edc7a34ad2032c4aa4aa